### PR TITLE
fixed Lunchbreak Testcases

### LINKED ISSUE #65: iCloud remote used in mock test
https://github.com/Ovakefali13/buerro/blob/ed9fb0e3142c7808faf5d8c54174ce2532856acb/usecase/lunchbreak/lunch.py#L117

https://github.com/Ovakefali13/buerro/blob/ed9fb0e3142c7808faf5d8c54174ce2532856acb/usecase/lunchbreak/lunch.py#L92

@timozuther 

The CalService is instantiated with the iCloud Remote non-regarding of whether we are in a mock or non-mocking test. 

Also you might think about moving the setting of Mock remotes and services out of the class into the test. Even though it might require some setters, I feel like it shouldn't the classes responsibility to handle mocking.

### PR DESCRIPTION
**Pull request by timozuther**
close #65

### CODE REVIEW

File: usecase/lunchbreak/lunch.py [hunk 1]
```diff
@@ -18,17 +19,40 @@ class Lunchbreak:
     start = None
     end = None
 
+    calender_remote = None
+
     def __init__(self, mock:bool=False):
         if mock:
+            # self.set_WeatherApapter(WeatherMock())
```
Comment (Ovakefali13) [inline usecase/lunchbreak/lunch.py]: Are these comments necessary ?

Comment (timozuther) [reply usecase/lunchbreak/lunch.py]: it is to be discussed if this is the future implementation

Comment (Ovakefali13) [reply usecase/lunchbreak/lunch.py]: I guess that's something we need to discuss with all @mariushegele @fionnfuchs @JannisTriesToCode what do you guys think?

**Comment by Ovakefali13** [comment]
@timozuther are you able to resolve these merge conflicts ?

**Pull request merged**